maptivist/src/main/webapp/script.js

```diff
@@ -79,8 +79,13 @@ var GoogleAuth;
       $('#sign-in-or-out-button').click(function() {
         handleAuthClick();
       });
-      $('#revoke-access-button').click(function() {
-        revokeAccess();
+
+      $('#myButton').click(function() {
```

Comment:
If I understand the code correctly, at first, both buttons says "Sign In" (or something similar). And then after the user signs in:

setSignInStatus() changes the text of the top "Sign In" button to  "Sign out" and the lower one becomes "Become a maptivist".

Then afterwards in this section, the button callbacks are changed to match ("Sign out" gets the sign out function and the "become a maptivist" get the open form function?)

I think the code would be clearer if it were organized such that the button callbacks and names were changed in the same place.

Reply:
You got the functionality right, I realized now that I was so focused on getting it to work I forgot to actually make it readable for other people. Ill work on that now. As for pulling from master, I use git pull.

Reply:
I managed to fix the issue by using GitHub Desktop on my personal PC and merging it through there, then pulling on my work laptop afterwards. This is unusual as I am always able to use "git pull".